modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -35,5 +35,13 @@
 		],
 		"newReference": "HtmlParser _htmlParser",
 		"to": "_htmlParser.extractText"
+	},
+	{
+		"from": "PhoneLocalServiceUtil.addPhone(long, String, long, String, String, long, boolean)",
+		"newImports": [
+			"com.liferay.portal.kernel.service.ServiceContextThreadLocal"
+		],
+		"issueKey": "LPS-193473",
+		"to": "PhoneLocalServiceUtil.addPhone(param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, param#6#, ServiceContextThreadLocal.getServiceContext())"
```

Comment:
@alberttgomes can we add `PhoneLocalServiceUtil` to the `classeNames` attribute and avoid this repetition on the `from` and `to`?

Reply:
Will do!!